rotkehlchen/chain/ethereum/modules/nft/nfts.py

```diff
@@ -280,36 +284,48 @@ def get_nfts_with_price(
             query_str += ' AND is_lp=?'
             bindings.append(lps_handling == NftLpHandling.ONLY_LPS)
 
+        if only_with_manual_prices is True:
```

Comment:
the filter is getting complicated here. Can't you build a normal filter like we do in the rest of the code?